geodiff/src/geodiff.cpp

```diff
@@ -1060,3 +1060,8 @@ bool GEODIFF_CT_columnIsPkey( GEODIFF_ChangesetTableH tableHandle, int i )
 {
   return static_cast<ChangesetTable *>( tableHandle )->primaryKeys.at( i );
 }
+
+const char *GEODIFF_G_getWkb( const char *gpkgWkb )
```

Comment:
I would call it: GEODIFF_createWkbFromGpkgHeader ; similarly the python function. Can you please improve docs in geodiff.h to exactly specify what is does , maybe with example?